db/models/models.js

```diff
@@ -46,3 +57,16 @@ exports.selectArticleComments = (article_id) => {
   });
 };
 
+
+
+
+
+exports.insertComment = (body, article_id) => {
+	return db
+		.query(`INSERT INTO comments (author, body, article_id) VALUES ($1, $2, $3) RETURNING*;`,
+			[body.username, body.body, article_id])
+		.then((comment) => {
+			return comment.rows;
```

Comment:
We should only return the element at index position 0 here as we only want the single result